x-pack/solutions/observability/plugins/streams_app/public/components/streams_list/index.tsx

```diff
@@ -0,0 +1,277 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import {
+  EuiBadge,
+  EuiButtonEmpty,
+  EuiButtonIcon,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiLink,
+  EuiSwitch,
+  EuiTitle,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React, { useMemo } from 'react';
+import { euiThemeVars } from '@kbn/ui-theme';
+import { css } from '@emotion/css';
+import { StreamDefinition, isWiredStream } from '@kbn/streams-schema';
+import { AbortableAsyncState } from '@kbn/observability-ai-assistant-plugin/public';
```

Comment:
we can import this one from `@kbn/observability-utils-browser` instead (they're dupes because of dependency hell)